.travis.yml

```diff
@@ -28,7 +28,7 @@ env:
     #- TARGET_PLATFORM=ubuntu/xenial
     - TARGET_PLATFORM=ubuntu/focal
 before_install:
-  - git clone -b v0.7.17 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b v0.7.18 --depth 1 https://github.com/citusdata/tools.git
```

Comment:
I don't see any changes that are relevant to this branch. Why did you want to bump the tools version?

It's totally ok, I am just curious

Reply:
yeah, just I didn't want to lose the track